frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx

```diff
@@ -80,10 +43,15 @@ const SelectCapacityAndNodes: React.FC<SelectCapacityAndNodesProps> = ({
 }) => {
   const { t } = useTranslation();
 
+  React.useEffect(() => {
+    dispatch({ type: 'capacityAndNodes/capacity', payload: '2Ti' });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Please add a comment regarding why we are disabling this rule.

Reply:
See the empty brackets on below line . We want to render it only first time.